react/src/pages/ServingListPage.tsx

```diff
@@ -288,6 +288,16 @@ const ServingListPage: React.FC<PropsWithChildren> = ({ children }) => {
                     <Typography.Text code>{endpoint_id}</Typography.Text>
                   ),
                 },
+                {
+                  title: t('modelService.ServiceEndpoint'),
+                  dataIndex: 'endpoint_id',
+                  render: (endpoint_id, row) =>
+                    row.url ? (
+                      <Typography.Text copyable>{row.url}</Typography.Text>
+                    ) : (
+                      <Tag>{t('modelService.NoServiceEndpoint')}</Tag>
+                    ),
+                },
```

Comment:
`Tag` is not good for showing messages.  It's better to use `-`.
```suggestion
                    row.url ? (
                      <Typography.Text copyable>{row.url}</Typography.Text>
                    ) :  '-'
                },
```

Reply:
I think so too.🤔 I will request review again after resolving your suggestion